app/main.py

```diff
@@ -1 +1,44 @@
-#  write your code here
+class Animal:
+
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
You don't need to use an `else` statement here. According to the checklist, it's better to write it like this:

python
if self.is_hungry:
    print(f"Eating {self.appetite} food points...")
    self.is_hungry = False
    return self.appetite
return 0
```

This way, the code is easier to read and understand.